### ISSUE TITLE
Started to translate looking_inside_neural_nets to zh_CN

### ISSUE THREAD

**Issue by weakish** [ml4a/ml4a.github.io#19]
I wish I finish it in tomorrow. ;-)

**Comment by genekogan** [comment]
fantastic! @weakish we also have another contributor who made the first chapter into Chinese, and she might already have some progress on that chapter. i can put you in contact with her. there is already another long chapter (how neural nets are trained).

**Comment by weakish** [comment]
@genekogan I did not setup GitHub notification emails. So I have already finished about 4/5 of translation upon noticing your reply yesterday.

I have read @tornoteli 's translation of the neural networks chapter, and enjoyed it.
Its translation has a high quality. I tried to convey the meaning of the original text as closely as possible, but I am afraid I can not reach the high standard of the previous chapter.
In other words, my translation is a little more loose.

**Comment by tornoteli** [comment]
Hi @weakish ! I'm glad you enjoyed. I enjoyed those chapters as well. Though after such a long time I think there may be lots of things to revise. If you want to revise it together please let me know. And I'm willing to collaborate anytime.

**Comment by weakish** [comment]
@tornoteli I will revise the first chapter this week and send a PR for it (and notify you to review my PR). ^_^

**Comment by tornoteli** [comment]
@weakish sounds great!
